android/src/test/java/com/onegini/mobile/sdk/reactnative/OneginiSdkWrapperTests.kt

```diff
@@ -181,4 +188,11 @@ class OneginiSdkWrapperTests {
 
         verify(authenticateUserUseCase).invoke("123456", "1", promiseMock)
     }
+
+    @Test
+    fun `when startSingleSignOn method is called calls startSingleSignOnUseCase with proper params`() {
```

Comment:
When startSingleSignOn method is called, Then startSingleSignOnUseCase is called with proper params

Reply:
Ignore this one as this useless test file has been removed in another commit.